packages/components/package.json

```diff
@@ -161,5 +167,11 @@
     "composed-offset-position": "^0.0.4",
     "lit": "^3.0.0"
   },
-  "web-types": "./dist/web-types.json"
+  "web-types": "./dist/web-types.json",
+  "meta": {
+    "checksums": {
+      "angular": "45415e7688194ab65f8f72efe2a27a658910c202",
+      "react": "00bfba6d406eb66250a884de281d565678cc0f0e"
```

Comment:
Question for later: When I add a new framework (e.g. vue), do I have to add this by hand or is this done via the script adjustments needed?

Reply:
It's automatically added.